syntax = "proto3";

package suggestions;

service Suggestions {
    rpc GetSuggestions (SuggestionsRequest) returns (SuggestionsResponse);
}

message SuggestionsRequest {
    string order_json = 1;
}

message SuggestedBook {
    string book_id = 1;
    string title = 2;
    string author = 3;
}

message SuggestionsResponse {
    repeated SuggestedBook books = 1;
}
